tests/features/steps/transducer.py

```diff
@@ -85,7 +87,7 @@ def step_impl(context):
 
 @then('the status should be one of "Draft", "Corrected"')
 def step_impl(context):
-    assert context.status in ("not reviewed",)
+    assert context.status in ("not reviewed",), f"Unexpected status: {context.status}"
```

Comment:
shouldn't this be `context.status in ("Draft", "Corrected")` to correspond with the the step definition?